test/k8s-integration/cluster.go

```diff
@@ -155,3 +155,33 @@ func downloadKubernetesSource(pkgDir, k8sIoDir, kubeVersion string) error {
 
 	return nil
 }
+
+func getGKEKubeTestArgs() ([]string, error) {
+	gkeEnv := "prod"
+	if len(os.Getenv("CLOUDSDK_API_ENDPOINT_OVERRIDES_CONTAINER")) > 0 {
+		gkeEnv = "staging"
+	}
+
+	cmd := exec.Command("gcloud", "config", "get-value", "project")
```

Comment:
I prefer the leave the project getting via gcloud regardless of in prow or now. At this point project for gcloud should be set to the correct value. I feel if its outside, it will be messier with at the very least another else clause that does the same code.